build.gradle

```diff
@@ -18,6 +18,7 @@ buildscript {
         classpath Dependencies.gradleNexusPublishPlugin
         classpath Dependencies.dokka
         classpath Dependencies.kotlinGradlePlugin
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.20'
```

Comment:
Why do we need this plugin here? Isn't `Dependencies.kotlinGradlePlugin` the same? 🤔